challenge.playground/Contents.swift

```diff
@@ -0,0 +1,69 @@
+import UIKit
+
+struct CountedSet<Element: Hashable> {
+    
+    private var dictionary: [Element: Int] = [:]
+    
+    mutating func insert(_ entry: Element) {
+        if let place = dictionary[entry] {
+            dictionary[entry] = place + 1
+        } else {
+            dictionary[entry] = 1
+        }
```

Comment:
```suggestion
       dictionary[entry, default: 0] += 1
```
Nothing wrong with your logic here but I thought I would share a short hand for the exact same logic, I actually tend to use this quite a bit, I find it very convenient so hopefully you will also.